lib/web/ui/app.go

```diff
@@ -121,7 +128,9 @@ type MakeAppsConfig struct {
 // MakeApp creates an application object for the WebUI.
 func MakeApp(app types.Application, c MakeAppsConfig) App {
 	labels := ui.MakeLabelsWithoutInternalPrefixes(app.GetAllLabels())
-	fqdn := utils.AssembleAppFQDN(c.LocalClusterName, c.LocalProxyDNSName, c.AppClusterName, app)
+	proxyPublicAddr := utils.InferProxyPublicAddr(c.RequestHost, c.ProxyPublicAddrs, c.LocalProxyDNSName)
```

Comment:
In this case, the value of `c.RequestHost` should always match one of the value in the `ProxyPublicAddrs` right? cause this request handled in the proxy itself.

Is it that if `AlwaysUseProxyPublicAddr` is true, can we directly use `proxyPublicAddr = c.RequestHost` ?